packages/website/docs/introduction.md

```diff
@@ -3,6 +3,39 @@ id: introduction
+
+For example, try typing the letter "s" into the **Search** bar on the top right of this site. You're given the ability to navigate directly to documentation on concepts like "**s**ources" and "**s**tate". If you continue to type, say "st", the results are refined. You see pages having to do with "**st**ate," "getting **st**arted," and "**st**atic sources."
+
+**Autocomplete is now a ubiquitous part of most search experiences.** Search providers like Google, e-commerce sites like Amazon, and messaging apps like Slack all offer autocomplete experiences on mobile and desktop.
+
+While this site's autocomplete shows documentation pages, autocomplete menus frequently show completed search terms. If you begin a search by typing the letter "j" into Google, it provides complete search terms like "javascript", "jest", "jsonlines", etc. Selecting one of these lands you on the search results page for the term.
+
+This experience minimizes typing, which is particularly impactful on mobile. It enables users to find what they're looking for quicker. It also exposes them to searches, products, or pages they may not have thought of but are interested in anyway.
+
+You may have already used UIs powered by this Autocomplete library. The [Algolia](https://www.algolia.com/doc/), [React Native](https://reactnative.dev/), and other documentation websites are powered by this library and the [DocSearch](https://docsearch.algolia.com/) project. This library is flexible enough to power more than just documentation search though. **Its purpose is to help you build interactive and accessible autocomplete experiences, regardless of your use case.**
+
+## What does this library provide?
+
+Autocomplete is a JavaScript library that lets you quickly build autocomplete search experiences. You can use it to create an autocomplete UX if you have:
+
+- Existing markup where you want to embed the experience,
+- Data to fill the autocomplete dropdown with.
+
+For example, if you have an element you want to embed an autocomplete experience into, you only need to configure it as the `container` in the [basic configuration options](/docs/basic-options). Then, once, you've configured the data to populate the autocomplete, the library creates and embeds the interactive DOM elements that make up the UX.
+
+The data that populates the autocomplete results are called [`sources`](/docs/sources). The library is agnostic as to what you use for your  [`sources`](/docs/sources).  They could be recent searches, a static set of searches terms, or from an external source, like an [Algolia](https://www.algolia.com/doc/guides/getting-started/what-is-algolia/) index.
+
+By configuring just those two required parameters (`container` and `getSources`) you can have an interactive autocomplete experience. **The library creates an input and provides the interactivity and accessibility attributes, but you're in full control of the DOM elements to output.**
```

Comment:
This sentence clashes a bit with what is said earlier:

> Then, once, you've configured the data to populate the autocomplete, the library creates and embeds the interactive DOM elements that make up the UX.

vs.

> The library creates an input and provides the interactivity and accessibility attributes, but you're in full control of the DOM elements to output.

We don't understand if Autocomplete provides the DOM elements or not.

Reply:
True, I don't think the first sentence adds much, and it's nice to move from the bullet point on data straight to "The data that populates the autocomplete results are called [`sources`](/docs/sources)."

Removed `once, you've configured the data to populate the autocomplete, the library creates and embeds the interactive DOM elements that make up the UX.` here: https://github.com/algolia/autocomplete.js/pull/393/commits/196f7a2781cdb536601a7c914863f95d1f63b36c